src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
1. Ideally, it would be better to put such variables in an env file
2. You create the same constant with the same url in all your api files, it should be the same common for all